src/es/components/organisms/header/nav-right-/nav-right-.html

```diff
@@ -3,6 +3,12 @@
         :host {
             background-color: transparent !important;
             min-height: 80px !important;
+            --header-nav-right-position: absolute !important;
+            top: 0 !important;
+            left: 0 !important;
+            right: 0 !important;
+            bottom: 0 !important;
+            width: 100% !important;
```

Comment:
This changes take effect on every page. For Home, it would be good if we used our own namespace, e.g. "home-top-" or "home-absolute-".